model/refArchitecture.js

```diff
@@ -0,0 +1,29 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var refArchSchema = new Schema({
+    name :{
+        type: String,
+        unique : false,
```

Comment:
Name must also be unique. There cannot be more than 1 reference architecture having **Full Stack** name, which causes ambiguity.